doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,171 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Catalogue                                               | A container for records. A catalogue is explicitly created and deleted by a user and has an identifier, assigned by the user.                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Cluster-wide object                                     | Components of a cluster-wide object are normally located on the servers of the same pool. For example, during the migration, a cluster-wide object can have a more complex layout with the components scattered across multiple pools.                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
I'd prefer to replace "cluster-wide" with "global" everywhere. Objects are not really "cluster-wide", they are striped within a particular pool.